packages/_vue3-migration-test/src/x-modules/search-box/components/test-search-box.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    <SearchInput />
```

Comment:
Can you configure these components trough props to test carefully everything is working?
Also, you can add some styles to position the placeholder over the input